components/b2b/AccountHierarchy/AccountHierarchyTree/AccountHierarchyTree.styles.ts

```diff
@@ -4,8 +4,8 @@ export const AccountHierarchyStyles = {
   expandCollapseButtonBox: {
     display: 'flex',
     justifyContent: { xs: 'flex-start', md: 'flex-end' },
-    marginTop: { xs: '-5px', md: '-50px' },
-    marginBottom: { xs: '10px', md: '20px' },
+    // marginTop: { xs: '-5px', md: '-50px' },
+    // marginBottom: { xs: '10px', md: '20px' },
```

Comment:
delete commented code if it's not required

Reply:
Done